src/api/posts.ts

```diff
@@ -0,0 +1,25 @@
+import { Post } from '../react-app-env';
+
+export const BASE_URL = 'https://mate.academy/students-api/';
```

Comment:
we can use this url, same as in comments api
```suggestion
export const BASE_URL_POSTS =`${process.env.BASE_URL}/posts`;
```